### DISCUSSION TITLE
Firmware updates blocked?

### DISCUSSION THREAD

**Discussion by robupham** [johanmeijer/grott#314] [Q&A]
I've been in discussion with Growatt tech support about an issue I have with my SPH inverter. Today they sent a firmware update to the inverter. Whilst I've seen evidence it restarted I don't believe the firmware update has actually applied correctly.

My guess is that grott (in proxy mode) hasn't passed through the firmware update. Would that be the expected behaviour? If so, is there any way for me to get it to allow firmware updates or should I temporarily reconfigure the Shine stick to not use my grott proxy?

For clarity, normal configuration updates when using the growatt site/app flow through fine.

Thanks.

**Comment by rany2** [comment]
In proxy mode if you did not set blockcmd it should just pass the request as is. There is no dropping of any command from Growatt unless blockcmd is set.

**Reply by robupham** [reply]
Thanks. That's what I assumed. However I wondered if they were trying to upload a large binary file (FW image) grott might not handle it.

**Reply by rany2** [reply]
It does handle it. In the past I've been able to update ~400KB worth of firmware to the datalogger and it flashed properly. (in proxy mode, clearly)

**Comment by johanmeijer** [comment]
I have the same experience. In proxy mode firmware updates are possible. But if you want to be sure it is not that difficult to revert back to a direct connection and download the firmware (via the growatt website).
